File: examples/crypto/price/node/getCryptocurrencyPrice.js
const { CryptoService } = require("m3o/crypto");

// Get the last price for a given crypto ticker
async function getCryptocurrencyPrice() {
  let cryptoService = new CryptoService(process.env.MICRO_API_TOKEN);
  let rsp = await cryptoService.price({
    symbol: "BTCUSD",
  });
  console.log(rsp);
}

getCryptocurrencyPrice();

File: examples/crypto/price/go/getCryptocurrencyPrice.go
package example

import (
	"fmt"
	"os"

	"github.com/micro/services/clients/go/crypto"
)

// Get the last price for a given crypto ticker
func GetCryptocurrencyPrice() {
	cryptoService := crypto.NewCryptoService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := cryptoService.Price(&crypto.PriceRequest{
		Symbol: "BTCUSD",
	})
	fmt.Println(rsp, err)
}
